src/View/Helper/NumberHelper.php

```diff
@@ -99,7 +53,7 @@ public function __call(string $method, array $params): mixed
      */
     public function precision(string|float $number, int $precision = 3, array $options = []): string
     {
-        return $this->_engine->precision($number, $precision, $options);
+        return Number::precision($number, $precision, $options);
```

Comment:
This proxying would also be covered by `__cal()` isn't it? I think we should remove all such methods which don't do anything extra and just add `@method` annotations instead for IDE autocompletion.

Reply:
Would the one-line comment for the method annotation just be "See Number::method()"?

I'm pretty sure we can specify defaults in annotations, but should check.

Reply:
Yeah such a short comment would be fine. Primarily the method signature is what's useful for IDEs.